### PR TITLE
Use logging module with GitHub Actions aware formatter

### PR DESCRIPTION
**Pull request by pavelzubarev**

### CODE REVIEW

**Review by ks93 — APPROVED** [review]
Lgtm

File: src/index.py [hunk 1]
```diff
@@ -25,7 +26,32 @@ def main(config: FunctionConfig) -> Optional[Function]:
...
 
+class GitHubLogHandler(logging.StreamHandler):
+    def __init__(self, stream=None):
+        super(GitHubLogHandler, self).__init__(stream=stream)
+
+    # https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-a-debug-message
+    def format(self, record):
+        message = super(GitHubLogHandler, self).format(record)
+        level_map: Dict = {
+            logging.CRITICAL: "warning",
```
Comment (haakonvt) [inline src/index.py:37]: Why not error here:
```py
logging.CRITICAL: "warning"
```

File: src/index.py [hunk 2]
```diff
@@ -52,4 +78,5 @@ def main(config: FunctionConfig) -> Optional[Function]:
     )
 
     if function is not None:
+        # return output parameter
         print(f"::set-output name=function_external_id::{function.external_id}")
```
Comment (haakonvt) [inline src/index.py:82]: Should this print-statement be changed to logging?

Comment (pavelzubarev) [reply src/index.py:82]: yes. that's how you return values out of action. magic syntax :)

Comment (haakonvt) [reply src/index.py:82]: love it

**Pull request merged**